.github/workflows/release-build-publish.yml

```diff
@@ -0,0 +1,58 @@
+name: Release build and publish
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to tag the release with, e.g., 1.2.0, 1.2.1-alpha.1
+        required: true
+
+jobs:
+  build_publish_daemon_image:
+    name: Build and Publish X-Ray daemon docker image to docker hub and public ECR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Login to Public ECR
+        uses: docker/login-action@v1
+        with:
+          registry: public.ecr.aws
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASS }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
```

Comment:
```suggestion
```

Reply:
I was wondering since in the alpha tag publish workflow we build binaries and copy it over so we don't have to build binaries in docker build push action. But in this workflow we probably have to build binaries in this workflow right (in docker build push action)? So wouldn't we require setup QEMU step?

Reply:
Oh I guess we haven't wired in build of artifacts here yet, in that case keeping qemu for now is fine, we'll remove after doing that.

Though we shouldn't release 3.3.0 until doing that :)